src/scripts/main.js

```diff
@@ -5,4 +5,17 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td>${person.name}</td>
+    <td>${(person.sex === 'm') ? 'Male' : 'Female'}</td>
+    <td>${person.born}</td> 
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${Math.ceil(person.died / 100)}</td>  
+  `;
+
+  document.querySelector('.dashboard').append(row);
```

Comment:
```suggestion
  dashboard.append(row);
```
`document.querySelector('.dashboard')` move this part out of loop